users/vieting/experiments/switchboard/hybrid/feat/specaug_sorted.py

```diff
@@ -159,8 +159,9 @@ def specaug_layer_sorted(in_layer):
         "class": "eval",
         "from": in_layer,
         "eval": "self.network.get_config().typed_value('specaugment_eval_func')("
-        "source(0, as_data=True, auto_convert=False), "
-        "network=self.network)",
+        "source(0, as_data=True, auto_convert=False),"
+        "network=self.network,"
+        "mask_divisor="+str(mask_divisor)+")",
```

Comment:
Btw, this breaks the hash for your previous setup, right? Since it will write the 5 here and that changes the hash.